cpu/cc26xx_cc13xx/include/cc26xx_cc13xx.h

```diff
@@ -28,29 +29,29 @@ extern "C" {
 typedef volatile uint8_t reg8_t;
 typedef volatile uint32_t reg32_t;
 
-/** @addtogroup CC26x0_cmsis CMSIS Definitions */
+/** @addtogroup CC13x2_cmsis CMSIS Definitions */
 /*@{*/
 
 /** interrupt number definition */
 typedef enum IRQn
 {
-    /******  Cortex-M3 Processor Exceptions Numbers ****************************/
+    /******  Cortex-M4 Processor Exceptions Numbers ****************************/
     ResetHandler_IRQn     = -15, /**<  1 Reset Handler                         */
     NonMaskableInt_IRQn   = -14, /**<  2 Non Maskable Interrupt                */
-    HardFault_IRQn        = -13, /**<  3 Cortex-M3 Hard Fault Interrupt        */
-    MemoryManagement_IRQn = -12, /**<  4 Cortex-M3 Memory Management Interrupt */
-    BusFault_IRQn         = -11, /**<  5 Cortex-M3 Bus Fault Interrupt         */
-    UsageFault_IRQn       = -10, /**<  6 Cortex-M3 Usage Fault Interrupt       */
-    SVCall_IRQn           = - 5, /**< 11 Cortex-M3 SV Call Interrupt           */
-    DebugMonitor_IRQn     = - 4, /**< 12 Cortex-M3 Debug Monitor Interrupt     */
-    PendSV_IRQn           = - 2, /**< 14 Cortex-M3 Pend SV Interrupt           */
-    SysTick_IRQn          = - 1, /**< 15 Cortex-M3 System Tick Interrupt       */
-
-    /******  CC26x0 specific Interrupt Numbers *********************************/
+    HardFault_IRQn        = -13, /**<  3 Cortex-M4 Hard Fault Interrupt        */
+    MemoryManagement_IRQn = -12, /**<  4 Cortex-M4 Memory Management Interrupt */
+    BusFault_IRQn         = -11, /**<  5 Cortex-M4 Bus Fault Interrupt         */
+    UsageFault_IRQn       = -10, /**<  6 Cortex-M4 Usage Fault Interrupt       */
+    SVCall_IRQn           = - 5, /**< 11 Cortex-M4 SV Call Interrupt           */
+    DebugMonitor_IRQn     = - 4, /**< 12 Cortex-M4 Debug Monitor Interrupt     */
+    PendSV_IRQn           = - 2, /**< 14 Cortex-M4 Pend SV Interrupt           */
+    SysTick_IRQn          = - 1, /**< 15 Cortex-M4 System Tick Interrupt       */
+
+    /******  CC13x2 specific Interrupt Numbers *********************************/
     EDGE_DETECT_IRQN      =   0, /**< 16 AON edge detect                        */
     I2C_IRQN              =   1, /**< 17 I2C                                    */
     RF_CPE1_IRQN          =   2, /**< 18 RF Command and Packet Engine 1         */
-    SPIS_IRQN             =   3, /**< 19 AON SpiSplave Rx, Tx and CS            */
+    PKA_IRQN              =   3, /**< 19 PKA interrupt                          */
```

Comment:
Are those really equal?

Reply:
I'm not sure where `SPIS` comes from. [cc13x2/cc26x2 TRM](http://www.ti.com/lit/ug/swcu185d/swcu185d.pdf) states it's PKA (p. 367).